spec/javascript/containers/ShowLocationContainerSpec.js

```diff
@@ -7,35 +7,24 @@ describe('Show location', () => {
   let routerProps
```

Comment:
This file seems to be a duplicate of /spec/javascript/containers/ShowLocation/Container?